.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015"],
```

Comment:
I think we'll want to use taskcluster's bable presets if we do this. I'll make a PR into this PR tonight with the changes for that since it's a bit confusing.